antora-playbook.yml

```diff
@@ -19,4 +19,8 @@ asciidoc:
     linkattrs: ''
     toc: ~
     quarkus_version: '2.9.1.Final'
-    required_maven_version: '3.8.1+'
\ No newline at end of file
+    quarkus_cli: https://quarkus.io/guides/cli-tooling
+    java_min_version: 11+
+    maven_min_version: 3.8.1+
+    graalvm_min_version: 22.0.0.2
+    kogito_examples: https://github.com/kiegroup/kogito-examples.git
```

Comment:
Are we using `_` or `-`? Your local variables are using `-`. I'm fine with whatever we decide, but we need to standardize, IMO.

Sorry for these late comments, but now that I'm writing another guide, I realized these small nits.

Reply:
`_` I suppose, will update it.